src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -220,8 +231,10 @@ public function addResource(ResourceInterface $resource)
     /**
      * {@inheritdoc}
      */
-    public function getMetadata($key = '', $domain = 'messages')
+    public function getMetadata($key = '', $domain = null)
     {
+        $domain = $this->getDefaultDomain($domain);
+
         if ('' == $domain) {
```

Comment:
I think it will break this call src/Symfony/Component/Translation/MessageCatalogue.php:170. 

https://github.com/lepiaf/symfony/blob/d41afca15eaed816231330dcef021050fd49415a/src/Symfony/Component/Translation/MessageCatalogue.php#L170